lib/msf/core/option_container.rb

```diff
@@ -528,7 +528,7 @@ def normalize(value)
   end
 
   def default
```

Comment:
Can you just delete this whole default method, since the base class 'OptBase' already defines the default attribute accessor?